valohai_yaml/objs/pipelines/parameter.py

```diff
@@ -0,0 +1,68 @@
+from typing import TYPE_CHECKING, List, Optional
+
+from valohai_yaml.lint import LintResult
+from valohai_yaml.objs.base import Item
+from valohai_yaml.objs.utils import check_type_and_listify
+from valohai_yaml.types import LintContext, SerializedDict
+
+if TYPE_CHECKING:
+    from valohai_yaml.objs import Config, Pipeline
+
+
+def get_pipeline_parameter_target_node(target: str) -> str:
+    return target.split(".", 1)[0]
+
+
+def get_pipeline_parameter_target_parameter_name(target: str) -> str:
+    return target.split(".", 1)[1]
+
+
+class Parameter(Item):
```

Comment:
I think this should be called PipelineParameter for clarity.

Reply:
I'm reluctant to include namespace in class name, it should be clear from the context. If we change this then I think we should name the other Parameter as ExecutionParameter. Of course this comes from the problem that **parameter** is very generic name for a class but I don't really know if there's a better name for it.

Reply:
I get where you're coming from with that, but `Parameter` is an entrenched term for execution parameters and task parameters (so renaming it to `ExecutionParameter` wouldn't necessarily be a good idea), and I can foresee it being very easy to accidentally import one `Parameter` when you mean the other `Parameter`...